test/test_dgmulti_3d.jl

```diff
@@ -135,6 +135,12 @@ isdir(outdir) && rm(outdir, recursive=true)
     )
   end
 
+  @trixi_testset "elixir_euler_tensorWedge.jl" begin
+    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_tensorWedge.jl"),
```

Comment:
Minor point, but I don't think we have CAMLCase in any of our elixir names. Could we change the name of the elixir to `elixir_euler_tensor_wedge.jl" instead?

Reply:
Done